ubuntudesign/www.ubuntu.com
# Phone devices

## Done

Restructured and styled the phone devices page.

## QA

Check `phone/devices` against the [copydoc](https://docs.google.com/document/d/1fuktWHXOV6iX5ZmdWzzHnkJoCfveca3mioQxeBYlJI8/edit)
